contrib/src/main/java/com/datatorrent/contrib/parser/CsvParser.java

```diff
@@ -62,247 +71,193 @@
  * @since 3.2.0
  */
 @InterfaceStability.Evolving
-public class CsvParser extends Parser<String, String>
+public class CsvParser extends Parser<byte[], KeyValPair<String, String>>
```

Comment:
This is enhancement to existing operator in the library. Not sure whether i can rename the operator itself.